src/SSHDebugPS/OutputWindow.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+using System;
+using System.Collections.Generic;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.SSHDebugPS
+{
+    class OutputWindow
+    {
+        private static OutputWindow _instance;
+
+        public static OutputWindow Instance
+        {
+            get
+            {
+                if (_instance == null)
+                {
+                    _instance = new OutputWindow();
+                }
+
+                return _instance;
+            }
+        }
+
+        private class PaneInfo
+        {
+            public PaneInfo(Guid paneId)
+            {
+                this.paneId = paneId;
+                this.Shown = false;
+            }
+
+            internal Guid paneId;
+            internal bool Shown { get; set; }
+        }
+
+        private IVsOutputWindow outputWindow;
```

Comment:
rethink the naming to differentiate between `this` object and the actual IVsOutputWindow.